debian/autobake-deb.sh

```diff
@@ -36,7 +36,7 @@ then
 
   # MCOL-4149: ColumnStore builds are so slow and big that they must be skipped on
   # both Travis-CI and Gitlab-CI
-  sed 's|-DPLUGIN_COLUMNSTORE=YES|-DPLUGIN_COLUMNSTORE=NO|' -i debian/rules
+  sed 's|$(CMAKEFLAGS)|$(CMAKEFLAGS) -DPLUGIN_COLUMNSTORE=NO|' -i debian/rules
   sed "/Package: mariadb-plugin-columnstore/,/^$/d" -i debian/control
```

Comment:
what is this for?

Reply:
With the change of the syntax this was modifying, we need now a new way to stop ColumnStore from building under Travis-CI and Salsa-CI, so I thought to sed in on the cmake line that echos out the CMAKEFLAGS.

Reply:
Without this both https://salsa.debian.org/mariadb-team/mariadb-server/-/pipelines/210408 and Travis-CI would fail. ColumnStore makes the build time 2-3x longer and is impossible to get passing on Travis-CI or Salsa-CI.